src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallDocumentServiceTest.kt

```diff
@@ -33,24 +32,6 @@ internal class RecallDocumentServiceTest {
     nomsNumber = NomsNumber("A1235B")
   )
 
-  @Test
-  fun `adding document throws 'recall not found' error if recall is missing`() {
-    every { recallRepository.getByRecallId(recallId) } throws EntityNotFoundException("boom!")
-
-    assertThrows<RecallNotFoundException> {
-      underTest.addDocumentToRecall(recallId, documentBytes, RecallDocumentCategory.PART_A_RECALL_REPORT)
-    }
-  }
-
-  @Test
-  fun `getting a document throws 'recall not found' error if recall is missing`() {
```

Comment:
QQ @mikedunfordEE : reasoning for deleting this test please?

Reply:
Not needed anymore.  The logic for throwing a RecallNotFoundException is in the RecallRepository and is tested there.  Don't think we need to test again here.  Possibly a service/component test or whatever could be added to document the behaviour of the endpoint if/when this ever happened.